airflow/models/taskinstance.py

```diff
@@ -3523,7 +3523,7 @@ def _schedule_downstream_tasks(
 
         except OperationalError as e:
             # Any kind of DB error here is _non fatal_ as this block is just an optimisation.
-            cls.logger().debug(
+            cls.logger().warning(
```

Comment:
According to the user report it is not something rare.

For the debugging purpose it could be enabled by [Advanced logging configuration](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/advanced-logging-configuration.html)

Reply:
That's one specific issue caused by dynamic task mapping. There are other issues pertaining to Celery that are more obscure. Having this log available outside of debug logs would help confirm whether the issue was related to the mini scheduler or not.

Reply:
FWIW we just switched this to debug, since we now use nowait: #38976
cc @dstandish

Reply:
Should the level stay at info and the message say something else then?

> Skipping mini-scheduling run because the scheduler has a lock on this DAG run

Reply:
Now that #39745 has merged, warning is probably appropriate now.